src/components/MyKiva/BlogCard.vue

```diff
@@ -66,6 +66,22 @@ const props = defineProps({
 	},
 });
 
+const optimizedImageUrl = computed(() => {
+	if (!props.image) return '';
+
+	// blog card dimensions: w:336px, h:55% = ~200px
+	const width = 336;
+	const height = 200;
```

Comment:
It doesn't look like height is needed, you can provide just width, so this can be removed.